src/OrchardCore.Modules/OrchardCore.Media/Views/Admin/Options.cshtml

```diff
@@ -72,4 +72,9 @@
         <code><input asp-for="AssetsPath" type="text" class="form-control-plaintext" /></code>
         <span class="hint">@T["The path used to store media assets. The path can be relative to the tenant's App_Data folder, or absolute."]</span>
     </div>
+    <label asp-for="AssetsUsersFolder" class="col-sm-2 col-form-label">@T["Assets users folder"]</label>
```

Comment:
I would not allow to change it through UI. Only from app settings.

Reply:
Or not at all, like we can't change `media`

Reply:
https://docs.orchardcore.net/en/latest/docs/reference/modules/Media/#configuration
media path can be change through appsettings.json, the UI only for display settings, can't save changes.